worker/echo/main.go

```diff
@@ -22,6 +24,12 @@ func echo(w *worker.Worker, addr string, id string, responseTo string, metadata
 		return fmt.Errorf("cannot call EmitEvent: %w", err)
 	}
 
+	// Sleep time between receiving the message and sending it
+	if sleepTime > 0 {
+		log.Infof("sleeping: %v", sleepTime.String())
```

Comment:
Using the format parameter `%v` will automatically call a type's `String()` method. There is no need to call it explicitly here.